recipe/meta.yaml

```diff
@@ -26,19 +21,29 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - make            # [unix]
-    - m2-gcc-libs     # [win]
-    - jom             # [win]
+
   host:
+    - bzip2
     - libcurl         # [unix]
     - expat           # [unix]
     - ncurses         # [unix]
+    - xz
+    - zlib {{ zlib }}
+    - libuv
     - rhash           # [unix]
+    - zstd {{ zstd }}
+
+  run:
     - bzip2
+    - libcurl         # [unix]
+    - expat           # [unix]
+    - ncurses         # [unix]
     - xz
     - zlib
-    - zstd
     - libuv
-
+    - rhash           # [unix]
+    - zstd
+    - vs2015_runtime  # [win]
```

Comment:
I'm curious, we specify vs2019 in cbc. Why is this 2015?

Reply:
@Arishamays1 I added the cbc on my path of figuring out the windows issues. VS2019 shouldn't be necessary, you can remove the `conda_build_config.yaml` file.

Reply:
Got it! Removed. Thanks for your help Eric!